xml/2018-07-25-barbecue.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<email title="DoCSoc Summer BBQ" agenda="false">
+
+  <text>
+    Hi everyone,
+
+    Hope your summer holidays are going well! With the great weather, DoCSoc
+    will be holding a summer barbecue in a couple weeks' time. If you're around
+    then, take a look at the event details below!
+  </text>
+
+  <hr/>
+
+  <event startDate="2018-08-12T13:00:00+01:00"
+    endDate="2018-08-12T17:00:00+01:00"
+    location="Prince's Gardens, SW7 2QJ">
+    <image src="https://i.imgur.com/l1GF0po.jpg" />
+    <text>
+      Working or staying in London over the summer? Come on down to the DoCSoc
+      summer BBQ for free food, drinks and laughs.
+
+      It will be held in Prince's Gardens from 1-5pm. Please click **Going** on
+      the Facebook event if you plan on coming so we have enough food for
+      everyone!
+
+      While there will be lots of food, if you would like to bring some extra
+      food or drinks, that would also be most welcome!
+
+      Note: you must be a member of DoCSoc to attend this event. Computing, JMC,
+      and EIE students are automatically DoCSoc members.
+    </text>
+    <links>
+      <link href="https://www.facebook.com/events/201855943797849/">Facebook</link>
```

Comment:
iff only the fb integration worked 🙈 
how's the progress on that?

Reply:
API doesn't seem to be able to retrieve non-subscribed events even with approval. Will have to investigate further.